core/cmd/jobs_commands.go

```diff
@@ -0,0 +1,182 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"github.com/urfave/cli"
+	"go.uber.org/multierr"
+)
+
+// JobRenderer wraps the JSONAPI Job Resource and adds rendering functionality
+type JobRenderer struct {
+	presenters.JobResource
+}
+
+// ToRows returns the job as a multiple rows per task
+func (j JobRenderer) ToRows() [][]string {
+	row := [][]string{}
+
+	// Produce a row when there are no tasks
+	if len(j.FriendlyTasks()) == 0 {
+		row = append(row, j.toRow(""))
+
+		return row
+	}
+
+	for _, t := range j.FriendlyTasks() {
+		row = append(row, j.toRow(t))
+	}
+
+	return row
+}
+
+// ToRow generates a row for a task
+func (j JobRenderer) toRow(task string) []string {
+	return []string{
+		j.ID,
+		j.Name,
+		j.Type.String(),
+		task,
+		j.FriendlyCreatedAt(),
+	}
+}
+
+// GetTasks extracts the tasks from the dependency graph
+//
+// TODO - Remove dependency on the pipeline package
```

Comment:
Does this need to be done before merge or will there be a followup?

Reply:
This is a general TODO but can remove if you prefer